app/src/main/java/com/cornellappdev/uplift/ui/components/gymdetail/GymTodaysClasses.kt

```diff
@@ -59,7 +63,8 @@ fun GymTodaysClasses(
         }
         if (todaysClasses.value.isEmpty()) {
             Text(
-                text = "We are done for today.\nPlease check again tomorrow!",
+                text = "Classes are coming soon!\nStay tuned for an Uplift update.",
+                // TODO: Change to: text = "We are done for today.\nPlease check again tomorrow!",
```

Comment:
can you not change the text right now?

Reply:
Not yet, cuz classes aren't in the backend yet. If I put it as text that indicates that there are no more classes today, that's a bit misleading. I'll change it back when the classes are added to backend.